.circleci/config.yml

```diff
@@ -69,7 +70,9 @@ jobs:
       - run:
           name: rails test
           command: |
-            bundle exec bin/rails test $(circleci tests glob "test/**/*_test.rb" | circleci tests split --split-by=timings)
+            TESTS=$(circleci tests glob "test/**/*_test.rb" | circleci tests split --split-by=timings)
+            echo $TESTS
+            bundle exec bin/rails test $TESTS
```

Comment:
```suggestion
            circleci tests glob "test/**/*_test.rb" | circleci tests run --command="xargs bundle exec rails test" --verbose --split-by=timings
```

Reply:
Yes, I'll do this also. It's just that I am trying to pinpoint the issue with tests timing out :confused: 
I didn't see it before, so I guess this is the result of Ruby upgrade. I hope it's just the tests, but would like to understand why it is happening, so trying to figure out whether it's a specific test that times out.